src/handfunc_macro.jl

```diff
@@ -41,18 +42,51 @@ macro func_vars(expr)
     return esc(Meta.quot(expr_numeric))
 end
 
-function handfunc(found_func, func_args, kwargs)
+function handfunc(found_module, found_func, func_args, kwargs)
     func_body = remove_return_statements(found_func.args[2])
     func_body = unblock(func_body)
     func_body = rmlines(func_body)
-    
+    func_body = _walk_func_body(func_body, found_module)
     kw_dict, pos_arr = _initialize_kw_and_pos_args(found_func, func_args)
     return_expr = _initialize_expr(kw_dict, pos_arr)
     push!(return_expr.args[2].args, :(@handcalcs $(func_body) $(kwargs...)))
     # ret = @eval eval_module $return_expr
     return return_expr
 end
 
+function _walk_func_body(expr::Expr, found_module)
+    found_module_sym = Symbol(found_module)
+    prewalk(expr) do x
+        if @capture(x, f_(args__))
+            len = length(collect(Leaves(f)))
+            if len == 1
+                if isdefined(Main, f) # should really be `Base` but `Main` is used for now
+                    x
+                else
+                    new_x = :($found_module_sym.$f($(args...)))
+                    new_x
+                end
+            else
+                _add_found_module!(f, found_module_sym)
+                new_x = Expr(:call, f, args...)
+                new_x
+            end
+        else
+            x
+        end
+    end
+end
+
+function _add_found_module!(ex, m)
+    if ex.args[1] isa Symbol
+        sym = ex.args[1]
+        ex.args[1] = Expr(:., m, QuoteNode(sym))
```

Comment:
This works if m is a symbol, but what if it is something like: `Foo.Bar`. Need to add additional logic